modules/apigee_edge_apidocs/apigee_edge_apidocs.links.menu.yml

```diff
@@ -0,0 +1,13 @@
+# API Doc menu items definition
+entity.apidoc.collection:
+  title: 'API Docs'
+  route_name: entity.apidoc.collection
+  description: 'List API Docs'
+  parent: system.admin_structure
```

Comment:
Would it be sensible to move API docs to content? It feels for me more like content, but maybe that's just me